package.json

```diff
@@ -10,20 +10,21 @@
   "license": "MIT",
   "main": "./index.js",
   "scripts": {
-    "coveralls": "nyc report --reporter=text-lcov | coveralls",
-    "coverage": "nyc report --reporter=html",
-    "test": "./node_modules/.bin/jshint * && nyc ./node_modules/.bin/_mocha ./test/*.js ./test/commands/*.js ./test/parser/*.js --timeout=8000",
-    "jshint": "./node_modules/.bin/jshint *"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "test": "nyc ./node_modules/.bin/_mocha ./test/*.js ./test/commands/*.js ./test/parser/*.js --timeout=8000",
+    "pretest": "optional-dev-dependency hiredis",
+    "posttest": "jshint ."
   },
   "devDependencies": {
     "coveralls": "^2.11.2",
-    "hiredis": "^0.4.1",
```

Comment:
newer versions of iojs and node.js have trouble with node-gyp + hiredis:

https://ci.appveyor.com/project/bcoe/node-redis/build/job/r1lcp1l8iyjfhw37
https://ci.appveyor.com/project/bcoe/node-redis/build/job/6774e4ym9ubmhi7u